Makefile

```diff
@@ -1,31 +1,27 @@
-help: ## Display this help screen
-	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
+.PHONY: wasm contracts keys circuits abi test run help
 
-wasm: ## Generate the WASM for the contract given (e.g. make wasm for=transfer)
-	@cargo rustc \
-		--manifest-path=contracts/$(for)/Cargo.toml \
-		--release \
-		--target wasm32-unknown-unknown \
-		-- -C link-args=-s
+wasm: ## Generate the WASM for all the contracts
```

Comment:
I believe since we're refactoring the Makefile, the best approach would generalize it, so for example if you want to keep the `wasm` recipe is fine, but instead of listing all the contracts we should just trigger `make` with the same recipe for the contracts subfolder. That will avoid to modify the Makefile for any possible contracts we're going to add.

I did something similar for `rusk-abi`: https://github.com/dusk-network/rusk/blob/master/rusk-abi/Makefile

The difference is that the contract is only used for test there, but the approach for the Makefile is the same.